### ISSUE TITLE
More low-hanging fruit: remove dplyr and ggplot2 from "Suggests:"

### ISSUE THREAD

**Issue by wlandau** [ropensci/drake#605]
Re https://github.com/ropensci/drake/issues/593#issuecomment-445100776, I have decided to deprecate `load_main_example()`. @bpbond, this is more low-hanging fruit. Steps:

- [ ] Move the `load_main_example()` function from `R/examples.R` to `R/deprecate.R`.
- [ ] As with other functions in `R/deprecate.R`, add a call to `.Deprecated()`.
- [ ] Modify the `roxygen2` comments:
    - [ ] Add `@keywords internal`.
    - [ ] Indicate the date of the deprecation happened in `@details`.
    - [ ] Refer users to `drake_example("main")`.
    - [ ] Shorten the other fields, such as `@examples`.
- [ ] Change [these two lines in the example's source](https://github.com/wlandau/drake-examples/blob/master/main/R/setup.R#L5-L6) to use `require()` rather than `library()`.
- [ ] Move the [test of `load_main_example()`](https://github.com/ropensci/drake/blob/3fc80ff304c5a92e4a2ff3791d14c40a2aae1760/tests/testthat/test-examples.R#L25-L42) to [`tests/testthat/test-deprecate.R`](https://github.com/ropensci/drake/blob/master/tests/testthat/test-deprecate.R) and invoke `expect_warning()` or `suppressWarnings()` as needed.
- [ ] Remove all calls to `skip_if_not_installed("dplyr")` and `skip_if_not_installed("dplyr")` in tests (the one in `test-visuals.R` does not actually need to be there.
- [ ] Remove `dplyr` and `ggplot2` from "Suggests:" in the `DESCRIPTION` file.

**Comment by bpbond** [comment]
>Remove dplyr and ggplot2 from "Suggests:"

What about `drake_ggraph`, though? That has numerous '::' imports from `ggplot2`.

**Comment by wlandau** [comment]
You are right, I forgot about that. I have not touched that part of the code by hand since the summer. Let's keep `ggplot2`. Revising the checklist.
